src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const priceRent = days * 40;
+  const shotrDiscount = 20;
+  const longDiscount = 50;
+
+  if (days < 3) {
+    return priceRent;
+  } else if (days >= 3 && days < 7) {
```

Comment:
```suggestion
  }
  
  if (days >= 3 && days < 7) {
```
no need in else if you have return inside previous `if`